src/ape_run/__init__.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+import click
+
+from ape.utils import Abort
+
+
+def _run_script(script_file, interactive=False):
+    # Load the python module and find our hook functions
+    ns: dict = {}
+    try:
+        py_module = compile(script_file.read_text(), str(script_file), "exec")
+        eval(py_module, ns, ns)
```

Comment:
Not sure what the tradeoffs would be but I think you can also do this step using `importlib`:

https://docs.python.org/3/library/importlib.html

Reply:
`importlib` seems clearer!